worker-caf/src/main/java/com/hpe/caf/worker/AbstractWorkerFactory.java

```diff
@@ -164,24 +164,27 @@ private T verifyWorkerTask(
      *
      * @param task the deserialised Worker task
      * @param tracking additional fields used in tracking task messages
+     * @param workerTaskData The worker task data to use during operation
      * @return the worker instance
      * @throws TaskRejectedException if a Worker cannot be created to handle this task currently
      * @throws InvalidTaskException if it appears this task cannot possibly be handled by a Worker of this type
      */
-    protected Worker createWorker(final T task, TrackingInfo tracking) throws TaskRejectedException, InvalidTaskException
+    protected Worker createWorker(final T task, final TrackingInfo tracking, final WorkerTaskData workerTaskData)
+        throws TaskRejectedException, InvalidTaskException
     {
-        return createWorker(task);
+        return createWorker(task, workerTaskData);
     }
```

Comment:
Since you're making breaking changes there is no point in keeping this overload.  (The tracking information can be retrieved from the worker task data).